install.sh

```diff
@@ -4,7 +4,7 @@
 # activecountermeasures.com
 
 # CONSTANTS
-_RITA_VERSION="v1.0.3"
+_RITA_VERSION=$(cat Version)
```

Comment:
Is there an advantage to switching this to a Version file or was this a temporary change?

Reply:
`install.sh` is written such that it can be run standalone without cloning the whole rita repo.  Our official install instructions are:
> Download the latest install.sh file from the release page
and this change would cause the install to fail since the `Version` file doesn't exist.

I think I need to add some comments about this in the script since it's not obvious why the version number is hard-coded.

Reply:
I took another look around for a way to get the version number and found github's go api. I'm writing a function that should be able to avoid this change.